packages/client/src/components/Main/Main.css

```
@@ -5,7 +5,7 @@ main {
       rgba(28, 65, 64, 0.8) 0.01%,
       rgba(7, 27, 32, 0.5) 100%
     ),
-    url('../../../public/assets/images/body-background-image-concrete.png');
+    url('../../../public/assets/images/concrete-bw.png');
```

```suggestion
    url('/assets/images/concrete-bw.png');
```

Once the app is published the URLs for assets need to be relative to the web sever.

---

@miltonhultgren Sorry I was so fast-forward I pressed this damn button. Could we undo it here? 😵‍💫

---

Could you make an opposite suggestion so that I can click it and merge the thing?